src/cephadm/cephadm

```diff
@@ -26,7 +26,7 @@ import errno
 import struct
 import ssl
 from enum import Enum
-
+from textwrap import dedent
```

Comment:
`cephadm:29:1: F401 'textwrap.dedent' imported but unused`

I think you removed your usage of this so the import will need to be removed as well

Reply:
Good catch! thanks :+1: